samples/operate-first/Makefile

```diff
@@ -17,6 +19,8 @@ clean: clean
 .PHONY: helm-template
 helm-template: $(TOOLBIN)/yq $(TOOLBIN)/helm
 	$(TOOLBIN)/helm template $(CRD_RELEASE_NAME) -n $(NAMESPACE) $(CRD_CHART_PATH) > fybrik-crd.yaml
+	$(TOOLBIN)/yq -i e '.plugins.vaultPluginSecretsKubernetesReader.clusterScope |= false | .plugins.vaultPluginSecretsKubernetesReader.namespaces |= . + ["fybrik-applications"] | .plugins.vaultPluginSecretsKubernetesReader.namespaces |= . + ["fybrik-system"]' $(VAULT_CHART_PATH)/values.yaml
+	$(TOOLBIN)/helm template $(VAULT_RELEASE_NAME) --values $(VAULT_CHART_PATH)/env/dev/vault-single-cluster-values.yaml -n $(NAMESPACE) $(VAULT_CHART_PATH) | $(TOOLBIN)/yq e 'select(.kind=="MutatingWebhookConfiguration" or .kind=="ClusterRole" or .kind=="ClusterRoleBinding")' -> fybrik-vault.yaml
 
```

Comment:
@eletonia can you please add --set "vault.injector.enabled=false" --set "vault.server.dev.enabled=true" same as in the quick start.

Reply:
Added